appinterface/projection/crossfire/crossfire.go

```diff
@@ -472,6 +472,12 @@ func (projection *Crossfire) checkTaskKeepActive(
 	}
 
 	for _, validator := range validators {
+		// If validator missed the task 1, it cannot complete the active task
+		if validator.TaskPhase1NodeSetup == constants.MISSED {
+			projection.logger.Debugf("Won't handle task keepActive for %s because it missed taskSetup", validator.OperatorAddress)
```

Comment:
This Log goes in the chain-indexing logs. 
Is that behaviour fine?

Reply:
I think it should be fine